runtime/jvmti/jvmtiExtensionMechanism.c

```diff
@@ -3807,9 +3807,7 @@ jvmtiGetVirtualThread(jvmtiEnv* jvmti_env, ...)
 		if (JVMTI_ERROR_NONE == rc) {
 			j9object_t threadObject = targetThread->threadObject;
 			j9object_t carrierThreadObject = targetThread->carrierThreadObject;
-			if ((NULL != threadObject)
-			&& (threadObject != carrierThreadObject)
-			) {
+			if ((NULL != threadObject) && (threadObject != carrierThreadObject)) {
```

Comment:
This looks like only a formatting change - was that the intent?

Reply:
Yes, only formatting was updated. The logic is unchanged.